.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  presets: [
+    ["@babel/preset-env", { targets: { node: true }}],
+    // The "all" property is needed for Flow to properly parse the syntax. It has
+    // trouble understanding when the flow comment is the second comment.
+    // See: https://github.com/babel/babel/issues/9845
+    ["@babel/preset-flow", { all: true }]
+  ],
+  plugins: [
+    // Without `loose`, the transformation uses `Object.defineProperty` which
+    // obeys the spec more, but we get a bundle that's 8kB larger. That was the
+    // default in babel v6.
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+    ["@babel/plugin-proposal-object-rest-spread", { useBuiltIns: true }]
+  ]
+}
```

Comment:
There's no "test" specific configuration anymore: we should have the same set-up now... although I haven't configured tests yet so maybe I'll have to bring it back.